src/data_scraper/time_helpers.py

```diff
@@ -68,4 +65,76 @@ def str_to_timestamp(string, format: ['date', 'exact_time']):
     elif format == 'exact_time':
         return int(round(dt.datetime.strptime(string, "%Y-%m-%d %H:%M:%S").timestamp())) * 1000
     else:
-        raise Exception('Format has to be either "date" or "exact_time".')
\ No newline at end of file
+        raise Exception('Format has to be either "date" or "exact_time".')
+
+
+def get_start_of_the_day(timestamp):
```

Comment:
I load full days of training data now starting from 00:00 on the start day and ending at 23:59 on the end day

Reply:
This will avoid any errors with missing data during the day